client/python/lib/test/config.py

```diff
@@ -14,6 +14,29 @@
 # limitations under the License.
 #
 
+from tensorflow_serving.apis.get_model_status_pb2 import GetModelStatusResponse, ModelVersionStatus
+from tensorflow_serving.util.status_pb2 import StatusProto
+
+def generate_response(times):
+    raw_response = GetModelStatusResponse()
+    model_versions = []
+
+    for i in range(times):
+        status = StatusProto()
+        status.error_code = 1
+        status.error_message = "msg"
+
+        model_version_status = ModelVersionStatus()
+        model_version_status.version = i
+        model_version_status.state = 10
+        model_version_status.status.CopyFrom(status)
+
+        model_versions.append(model_version_status)
+    raw_response.model_version_status.extend(model_versions)
+    return raw_response
+
+MODEL_RESPONSE_CORRECT = [(generate_response(i), i) for i in range(4)]
```

Comment:
I think it's not intuitive to evaluate what the test cases will be. Also generated state and error message is hardcoded.
Maybe we could have two methods:
- create_model_status_response - that would take version, code and error message as arguments
- merge_model_status_responses - that would take a list of objects generated by the method above and merge them into one

This way the specification of the single version response can be easily read from the function parameters and we could create more complex responses (with multiple version in different states with different messages) in a more readable form.